feature/platform/controllercard/tests/SetRequest_controll_card_switchover/README.md

```diff
@@ -0,0 +1,34 @@
+# gNMI-1.19 ConfigPush after Control Card switchover
+
+## Summary
+This test verifys if large config can be bushed via gNMI SetRequest within 2 minutes after Control Card switchover. 
+
+## Procedure
+
+* Prepare large OpenConfig config file to be pushed within single setRequest.
+  * 150 LAG interfaces w/ ip, ipv6 configuration
+  * 800 Ethernet interfaces as memeber s of LAG
+  * 28 IPv4 and 28 IPv6 BGP neighbours
+  * ISIS on all trunk/LAG ports
+* Initiate Control Card switchover using gNOI SwitchControlProcessorRequest 
+* Immedietly after reciving SwitchControlProcessorResponce for  gNOI switchover, but no later then 120 second after calling gNOI SwitchControlProcessorRequest, send gNMI setRequest with prepared large config.
```

Comment:
Is there any wait time allowed after receiving `SwitchControlProcessorResponse` (within the mentioned 120 seconds) before sending gnmi set?

Reply:
No. There is not allowed wait time after receiving `SwitchControlProcessorResponse`. The 120s timeout is here to cover case when `SwitchControlProcessorResponse` is not received.

Reply:
One of the goals of this test is to validate that the DUT sends an appropriate response to a `Set` during time that the switchover is occurring.    @rszarecki perhaps this logic covers the possibility of an error message and incorporates the 2 minute deadline:

If a gRPC error is returned, the client should re-try the gnmi.Set every 10 seconds.
After 120 seconds, if no successful setResponse is received, the test should be marked as a failure.

Reply:
@dplore PTAL

Reply:
LGTM